README.md

```diff
@@ -76,7 +76,7 @@ OpenMRS [BDD](https://en.wikipedia.org/wiki/Behavior-driven_development) QA fram
 - `mvn clean install -DskipTests=true`
 
 ## Configuration
-Set Your test configurations in `qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test.properties`.
+Set your test configurations in `qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test.properties`.
```

Comment:
Hey @kdaud point of clarification. Did we confirm that our tests are breaking from upgrade of platform version. My take  is it possible to revert back our platform version we depending on and simply depend on the version that is working for us. cc @dkayiwa

Reply:
@sherrif10 the solution is to fix the tests because the platform has no problem

Reply:
FYI: https://github.com/openmrs/openmrs-contrib-qaframework/commit/13e8266bc24843f304d930a279a3c5a80a378277

Reply:
@dkayiwa  thanks for the quick fix. that seemed out of scope. Now lets embark on why tests are failing on ci.

Reply:
https://issues.openmrs.org/browse/REPORT-886

Reply:
Yes @dkayiwa Thanks that is a fantastic quick fix.